proto/iot.proto

```diff
@@ -165,6 +165,24 @@ message Meter {
     Double apparent_power = 3 [(brick_point_class).namespace='https://brickschema.org/schema/1.0.3/Brick#brick', (brick_point_class).value='Apparent_Power_Sensor'];
     //unit: KWh
     Double energy = 4 [(brick_point_class).namespace='https://brickschema.org/schema/1.0.3/Brick#brick', (brick_point_class).value='Energy_Sensor'];
+    //unit: kVAR
+    Double reactive_power = 4 [(brick_point_class).namespace='https://brickschema.org/schema/1.0.3/Brick#brick', (brick_point_class).value='Reactive_Power_Sensor'];
+    //unit: Amps
+    Double current = 5 [(brick_point_class).namespace='https://brickschema.org/schema/1.0.3/Brick#brick', (brick_point_class).value='Current_Sensor'];
+    //unit: kW
+    Double demand = 6 [(brick_point_class).namespace='https://brickschema.org/schema/1.0.3/Brick#brick', (brick_point_class).value='Demand_Sensor'];
+    //unit: Gallons
+    Double water_total = 6 [(brick_point_class).namespace='https://brickschema.org/schema/1.0.3/Brick#brick', (brick_point_class).value='Water_Total_Sensor'];
```

Comment:
Each field needs to be numbered uniquely

Reply:
Additionally (and this is a bug in the repo as well), the namespace should end in `Brick#` rather than `Brick#brick`:

e.g.

```proto
Double water_total = 6 [(brick_point_class).namespace='https://brickschema.org/schema/1.0.3/Brick#', (brick_point_class).value='Water_Total_Sensor'];
```

Reply:
Oops! Ah just noticed this, I will make sure to look at the Travis test results and checks in the future, thanks :)

Reply:
Ok! Changed the numbering to be unique. I also changed all the Brick#brick instances to Brick# (within the Meter and Light proto messages). The Travis CI tests have yet to finish at the moment of this comment but I will make sure to correct any remaining issues if any.